lib/utils/pxclient.js

```diff
@@ -64,7 +63,7 @@ function sendToPerimeterX(activityType, details, pxCtx) {
 function submitActivities() {
     pxlogger.debug('Sending activities to perimeterx servers');
 
-    const tempActivities = _.cloneDeep(activitiesBuffer);
+    const tempActivities = activitiesBuffer.concat();
```

Comment:
someone showed me that `slice(0)` might be a bit more effective as a cloning method :)